packages/react-charting/src/components/DonutChart/Arc/Arc.styles.ts

```diff
@@ -29,5 +29,32 @@ export const getStyles = (props: IArcProps): IArcStyles => {
         },
       },
     },
+    tooltip: {
+      ...theme.fonts.medium,
+      display: 'flex',
+      flexDirection: 'column',
+      padding: '8px',
+      position: 'absolute',
+      textAlign: 'center',
+      top: '0px',
+      background: theme.semanticColors.bodyBackground,
```

Comment:
is there a default theme always available

Reply:
yes, the theme is coming as a prop from the parent component, which is a required property